backport-changelog/6.9/63250.md

```diff
@@ -0,0 +1,3 @@
+https://github.com/WordPress/wordpress-develop/pull/8663
+
+* https://github.com/WordPress/gutenberg/pull/69856
```

Comment:
Changes in the block-library directory are automatically synced to core, so no manual backporting is necessary.

Reply:
@t-hamano Thanks for the review! I've removed my last commit as requested.